From 79cf34e7324aaf2411e1a0d4c9cbd58047da7cce Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Wed, 15 Aug 2018 10:25:27 +0800
Subject: [PATCH] arm64: dts: rockchip: Remove initial rate of npll for px30

As npll rate may be changed when enable vopl in uboot, so we can't
change npll rate in kernel on px30.

Change-Id: If62da5bb77cdd411a550b2dc6250d654134474e3
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/px30.dtsi | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/px30.dtsi b/arch/arm64/boot/dts/rockchip/px30.dtsi
index 12b302a2c360..c67d27b94728 100644
--- a/arch/arm64/boot/dts/rockchip/px30.dtsi
+++ b/arch/arm64/boot/dts/rockchip/px30.dtsi
@@ -1008,9 +1008,6 @@
 		rockchip,boost = <&cpu_boost>;
 		#clock-cells = <1>;
 		#reset-cells = <1>;
-
-		assigned-clocks = <&cru PLL_NPLL>;
-		assigned-clock-rates = <1188000000>;
 	};
 
 	cpu_boost: cpu-boost@ff2b8000 {
-- 
2.35.3

